docs/public/_redirects

```diff
@@ -196,6 +196,18 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /zh/api/time-picker/ /zh/x/api/date-pickers/time-picker/ 301
 /pt/api/time-picker/ /pt/x/api/date-pickers/time-picker/ 301
 
+### Group & Pivot page split https://github.com/mui/mui-x/pull/4441
+/x/react-data-grid/group-pivot#tree-data /x/react-data-grid/tree-data 301
+/x/react-data-grid/group-pivot#master-detail /x/react-data-grid/master-detail 301
+/x/react-data-grid/group-pivot#row-grouping /x/react-data-grid/row-grouping 301
```

Comment:
We miss the leading slash to avoid a double redirection

```suggestion
/x/react-data-grid/group-pivot/#row-grouping /x/react-data-grid/row-grouping/ 301
```

Reply:
Hmm, I don't get this one. Why is slash needed here?

Reply:
Because this is how Next.js is configured to output HTML files. We don't output /x/react-data-grid/group-pivot.html but /x/react-data-grid/group-pivot/index.html. Which is shown as `/x/react-data-grid/group-pivot/`.